datasync/chaindatafetcher/kas/repository_transactions.go

```diff
@@ -48,10 +48,9 @@ var TxFilteringTypes = map[types.TxType]bool{
 }
 
 // transformToTxs transforms a chain event to transaction list according to the KAS database scheme.
-func transformToTxs(event blockchain.ChainEvent) []*Tx {
+func transformToTxs(event blockchain.ChainEvent) ([]*Tx, map[common.Address]struct{}) {
 	var txs []*Tx
-	// TODO-ChainDataFetcher extract EOAs in order to expire the KAS cache
-	//expiredEOAs := make(map[common.Address]struct{})
+	expiredEOAs := make(map[common.Address]struct{})
```

Comment:
How about changing `expiredEOAs` to `updatedEOAs` or something else? I can understand the expression that txs expire EOAs.

Reply:
Updated. 38e60923eeea50bf311ce328a6f1ddc1ff49a3d5